clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -76,6 +79,30 @@ class ServerGroupController {
     serverGroup
   }
 
+  @RequestMapping(value = "/serverGroupsDetailsForList", method = RequestMethod.POST)
```

Comment:
I don't love the name or that I have to use a POST, but I really don't want to have to batch tiny GET requests from deck since, e.g., when we do INT stack searches, we could be passing 100+ strings of decent length.  Open to better suggestions here.